resource_customizations/apigee.cloud.google.com/ApigeeDatastore/health.lua

```diff
@@ -0,0 +1,13 @@
+health_status = {}
+if obj.status ~= nil then
+  if obj.status.state ~= nil then
+    if obj.status.state == "running" then
+      health_status.status = "Healthy"
+      health_status.message = "Install of Apigee Datastore is done"
+      return health_status
+    end
+  end
+end
+health_status.status = "Progressing"
+health_status.message = "An install plan for an Apigee Datastore is pending installation"
```

Comment:
Let's generalize this so we can put all the checks into a single file:
```suggestion
health_status.message = string.format("Waiting for %s", obj.kind)
```